spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/procedures/BaseProcedure.java

```diff
@@ -201,7 +201,7 @@ TableCatalog tableCatalog() {
 
   /**
    * Closes this procedure's executor service if a new one was created with {@link
-   * #executorService(int, String)}. Does not block for any remaining tasks.
+   * BaseProcedure#executorService(int, String)}. Does not block for any remaining tasks.
```

Comment:
And can you also fix for other spark versions aswell?

Reply:
I haven't compiled myself. But why it is not giving the same warnings for other javadocs in the same class, like `#withIcebergTable`?